### DISCUSSION TITLE
no metrics by uri /metrics

### DISCUSSION THREAD

**Discussion by boleeinoe** [wg-easy/wg-easy#1453] [Q&A]
I downloaded the latest version of the repository, changed docker-compose.yml to receive metrics for prometheus, but there are none on the url http://ip:51821/metrics, it returns 404 (not found), what to do

**Comment by boleeinoe** [comment]
I seem to have found the answer that this is not yet available in latest, if so, please write about it on the repository page itself.

Also please make it possible to access the API with white cookies

**Comment by boleeinoe** [comment]
when do you plan to bring functionality for collecting metrics to latest?

**Comment by sky-py** [comment]
the same. 
no metrics by uri /metrics/json
